docs/_reference/RTMClient.md

```diff
@@ -26,8 +26,9 @@ permalink: /reference/RTMClient
     * [.handleWsMessage(wsMsg)](#RTMClient+handleWsMessage)
     * [._handleMessageAck(replyTo, message)](#RTMClient+_handleMessageAck)
     * [.handleWsError(err)](#RTMClient+handleWsError)
-    * [.handleWsClose()](#RTMClient+handleWsClose)
```

Comment:
were there arguments that were previously undocumented, or actually no arguments?